deploy/infrastructure/postgres/development/README.md

```
@@ -0,0 +1,53 @@
+ \l
+
+# See all users/roles
+ \du
+```
+
+```bash
+ # Connect to leaderboardstat database
+ docker exec -it rankr-rankr-shared-postgres-1 psql -U leaderboardstat_user -d leaderboardstat_db
+```
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)